homeassistant/components/ohme/strings.json

```
@@ -32,6 +32,18 @@
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
   },
+  "services": {
+    "list_charge_slots": {
+      "name": "List charge slots",
+      "description": "Return a list of charge slots.",
+      "fields": {
+        "config_entry": {
+          "name": "Ohme",
+          "description": "The Ohme config entry for which to return charge slots."
```

```suggestion
          "name": "Ohme instance/device/account",
          "description": "The Ohme config entry for which to return charge slots."
```
I forgot which one applied